service/routes.py

```diff
@@ -43,6 +43,27 @@ def init_db():
     Recommendation.init_db(app)
 
 
+######################################################################
+# LIST RECOMMENDATIONS
+######################################################################
+@app.route("/recommendations", methods=["GET"])
+def list_recommendations():
+    """ Returns all of the Recommendations """
+    app.logger.info("Request for recommendations list")
+    recommendations = []
+    recommendation_type = request.args.get("recommendation_type")
+    active = request.args.get("active")
+    if recommendation_type:
+        recommendations = Recommendation.find_by_recommendation_type(recommendation_type)
+    elif active:
+        recommendations = Recommendation.find_by_active(active)
+    else:
```

Comment:
You can remove these lines because we don't have find_by_recommendation_type and find_by_active now.

Reply:
I made new commits to fix these. can you double check?